### PR TITLE
Englishization of documents

### PR DESCRIPTION
**Pull request by cptbtptpbcptdtptp**

### CODE REVIEW

File: docs/first-game.zh-CN.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ type: 入门
   - Bird （小鸟的帧动画）
```
[the remark this answers is not in the document]
Comment (cptbtptpbcptdtptp) [reply docs/first-game.zh-CN.md:34]: Done.

File: docs/first-game.md [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+
+<img src="https://intranetproxy.alipay.com/skylark/lark/0/2021/png/13456322/1623310879070-048cbc6d-2193-4672-9440-c9a13d31f4d8.png" alt="image.png" style="zoom:50%;" />
+
+- The game is a 2D game, using [Orthogonal Camera](${docs}camera-cn#正交投影) will be easier and more convenient.
+- The game is divided into three states:
+  - Idel
+  - In the game
+  - Game over
+- Display objects throughout the game (from far to near):
+  - Backgound
```
[the remark this answers is not in the document]
Comment (cptbtptpbcptdtptp) [reply docs/first-game.md]: Done.

File: docs/first-game.md [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+  - Pipe
+  - Ground
+  - Bird
+  - GUI
+
+After the requirements analysis, we can basically sort out the game process and the overall UI layout. The next step is to build the entire UI interface according to the hierarchical relationship.
```
[the remark this answers is not in the document]
Comment (cptbtptpbcptdtptp) [reply docs/first-game.md]: Done.

File: docs/first-game.md [hunk 4]
```diff
@@ -0,0 +1,440 @@
...
+
+### Frame animation
+
+Refer to the implementation of the example `sprite-sheetAnimation`, we add a `Script` component to the `Entity`, and control the rendering area of ​​the sprite in the `onUpdate` function of the `Script` The status of the bird (alive or dead) shows different behaviors:
+
+<playground src="sprite-sheetAnimation.ts"></playground>
+
+### tweenjs
+
+In order to realize the flying and falling of the bird, we introduce [tweenjs]](https://github.com/tweenjs/tween.js), when the bird is flying, rotate the bird counterclockwise to 20 degrees, and the bird falls At that time, the rotation angle of the easing bird is -90 degrees. Looking through the source code of the easing component, you can find that he updates the value through recursion. Developers who are familiar with [transform component] (${docs}transform-cn) will find this A recursive way to change the components in the coordinate information one by one cannot make the `Entity` change the position in real time.
```
[the remark this answers is not in the document]
Comment (cptbtptpbcptdtptp) [reply docs/first-game.md]: Done.

File: docs/first-game.md [hunk 5]
```diff
@@ -0,0 +1,440 @@
...
+
+<img src="https://intranetproxy.alipay.com/skylark/lark/0/2021/png/13456322/1625132333754-89ca4388-6af6-4ce7-b7c0-395e9abb48bf.png" alt="image.png" style="zoom:50%;" />
+
+Here you can refer to the [Ray casting](${docs}ray-cn) document. After obtaining the ray, we can directly use the Restart button to surround [bounds](${api}oasis-engine/SpriteRenderer#bounds) to intersect the ray Check [intersectBox](${api}oasis-engine/Ray#intersectBox) and notify GameCtrl to restart the game after a collision. Here I simply encapsulate a Touch component. We will improve the input capability of the engine at the next milestone.
+
+# Collision detection
+
+## Timing of collision detection
+
+There is a premise when performing collision detection, that is, it is already the final position of this frame. Developers who have studied [Oasis scripting system](${docs}script-cn#组件生命周期函数) should be familiar with it. , After we change the position of `Entity` in `onUpdata`, we can do collision detection in `onLateUpdate`, which can ensure that there is no problem with the timing.
```
[the remark this answers is not in the document]
Comment (cptbtptpbcptdtptp) [reply docs/first-game.md]: Done.

File: docs/first-game.md [hunk 6]
```diff
@@ -0,0 +1,440 @@
...
+      }
+      break;
+    }
+  }
+});
+```
+
+# Optimization
+
+We have sorted out all the objects and asked them to "perform their duties". Now we only need to add "billion points" details on it. Of course, after completing the entire process, you can also try to optimize the game more:
```
[the remark this answers is not in the document]
Comment (cptbtptpbcptdtptp) [reply docs/first-game.md]: Done.

**Review by yangfengzzz — APPROVED** [review]
+1

**Pull request merged**